### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Falana-G**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Hey this is my first repository
+Kaisa hai lodu!!
+I forked u babyy!!
```
Comment (Tejvansh1) [inline README.md:3]: shut upppp

Comment (aprambir4834) [reply README.md:3]: berryy truee

Comment (aprambir4834) [reply README.md:3]: hor sunao mundeyo ki haal ne

Comment (Falana-G) [reply README.md:3]: badhiya badhiya...

Comment (aprambir4834) [reply README.md:3]: kya scene hai aj ka ?

Comment (Falana-G) [reply README.md:3]: Lodu btaya nhi kaisa hai??!!

Comment (Tejvansh1) [reply README.md:3]: kutch khaas nhi

Comment (aprambir4834) [reply README.md:3]: Khaane mai kya layea hai Keerat?

Comment (Falana-G) [reply README.md:3]: loon vali roti

Comment (Tejvansh1) [reply README.md:3]: joote chapl

Comment (Falana-G) [reply README.md:3]: Oye lodu... Btaya nhi fir kaisa hai?!!

Comment (Falana-G) [reply README.md:3]: agla lec konsa hai?

Comment (aprambir4834) [reply README.md:3]: MP

Comment (aprambir4834) [reply README.md:3]: Screenshot (14)

Comment (Falana-G) [reply README.md:3]: Oye lodu... Jo report kre.....!!!!

Comment (aprambir4834) [reply README.md:3]: Capture

**Pull request closed without merging**